docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -75,7 +75,7 @@ spec:
 EOF
 ----
 
-* When `neutron-sriov-nic-agent` is running on the existing Compute nodes, check the physical device mappings and ensure that they match the values that are defined in the `OpenStackDataPlaneNodeSet` custom resource (CR). For more information, see xref:pulling-configuration-from-tripleo-deployment_reviewing-configuration[Pulling the configuration from a {OpenStackPreviousInstaller} deployment].
+* When `neutron-sriov-nic-agent` is running on the existing Compute nodes, check the physical device mappings and ensure that they match the values that are defined in the `OpenStackDataPlaneNodeSet` custom resource (CR). For more information, see xref:pulling-configuration-from-tripleo-deployment_adopt-control-plane[Pulling the configuration from a {OpenStackPreviousInstaller} deployment].
```

Comment:
Does 'when' mean 'if' here?

Also, is the word 'existing' necessary here. I assume this refers to the compute nodes of the RHOSP deployment